packages/extensions/openapi-to-typespec/test/arm-analysisservices/tsp-output/models.tsp

```diff
@@ -74,19 +74,6 @@ union ProvisioningState {
   Scaling: "Scaling",
 }
 
-/**
```

Comment:
Expected. We changed the behavior to setting common type version v3. so that we could leverage the models from arm library.